config.json

```diff
@@ -107,9 +107,15 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 2,
```

Comment:
If we change difficulty, don't we need to put it into correct order in this array ?